mvp_project/README.md

```diff
@@ -1,109 +1,183 @@
-- Интерактивный пользовательский интерфейс
-- Хранение данных на сервере
+---
 
-## Технологии:
-- **Frontend**: Astro
-- **Backend**: FastApi
-- **База данных**: Post
-- **Авторизация**: --
+## **Основные функции**
+1. **Регистрация и аутентификация пользователей**
+   - Регистрация через email.
+   - Вход с использованием email/пароля.
+   - Авторизация через социальные сети (Google, Facebook и другие).
 
-# User Stories для интерактивного дневника путешествий
+2. **Создание и управление записями**
+   - Добавление текстов, фото, видео и геометок.
+   - Редактирование и удаление записей.
 
-## Категория 1: Управление дневником путешествий
+3. **Маршруты и карты**
+   - Автоматическое или ручное добавление маршрутов.
+   - Визуализация точек путешествия на карте.
 
-### 1. Создание дневника
-- **Как пользователь**, я хочу создать новый дневник с названием, описанием и обложкой, чтобы структурировать свои путешествия.
-  - **Контекст:** Пользователь заходит в профиль, нажимает "Создать дневник", вводит название, добавляет обложку и описание.
-  - **Технически:** FastAPI обрабатывает POST-запрос с данными дневника и сохраняет их в базе PostgreSQL.
+4. **Социальные взаимодействия**
+   - Обмен дневниками через ссылки.
+   - Лента активности с обновлениями друзей.
 
-### 2. Добавление мультимедиа
-- **Как пользователь**, я хочу загружать фотографии и видео, чтобы мои записи выглядели живыми и визуально привлекательными.
-  - **Контекст:** После создания дневника пользователь добавляет медиафайлы и просматривает их предварительно.
-  - **Технически:** Загружаемые файлы отправляются на сервер через FastAPI и сохраняются в облачном хранилище (например, AWS S3).
+5. **Приватность и настройки**
+   - Настройка видимости записей: публичные, по ссылке, личные.
 
-### 3. Организация записей
-- **Как пользователь**, я хочу упорядочивать свои записи по дням и местам, чтобы видеть путешествие в хронологическом порядке.
-  - **Контекст:** Пользователь добавляет новые записи с указанием дат и местоположений.
-  - **Технически:** В базе данных каждая запись связывается с дневником через внешние ключи.
+6. **Геймификация**
+   - Достижения и награды за активность.
+   - Баллы за посещение новых мест.
 
-### 4. Автоматическое отслеживание маршрутов
-- **Как пользователь**, я хочу, чтобы приложение автоматически записывало мой маршрут с использованием GPS, чтобы избежать ручного ввода.
-  - **Контекст:** Пользователь включает GPS, и приложение фиксирует передвижения в реальном времени.
-  - **Технически:** FastAPI принимает данные о местоположении через WebSocket или периодические POST-запросы.
+7. **Премиум-функции**
+   - Расширенное хранилище.
+   - Уникальные шаблоны оформления.
+   - Генерация видеоисторий.
 
 ---
 
-## Категория 2: Социальные взаимодействия
+## **Технологии**
+- **Frontend**: ---
+- **Backend**: FastAPI  
+- **База данных**: PostgreSQL  
+- **Облачное хранилище**: AWS S3 (или аналог)  
+- **API для карт**: Google Maps API или Mapbox  

```

Comment:
For kz 2gis and yandex map might a good option.

but if your goal is international users then Google Maps